eclipse-theia/theia
# keybindings: search improvement

#### What it does
Closes #11873

This commit aims to improve the keybinding search results in the Keyboard Shortcut Widget.

#### How to test
Similar to the linked issue.
1. Open the keybinding widget
2. Search for a key combination, e.g. Ctrl+U
3. Check that the proper keybindings are shown.
##### Notes
Ctrl+U should show keybindings with `Ctrl` and `U`, with the following being examples that may or may not exist already.
- `Ctrl`+`U`
- `Ctrl`+`Alt`+`U`
- `Ctrl`+`T` `U`
- `Ctrl`+`K` `Ctrl`+`U`.

Other search terms that should work for finding `Ctrl`+`U` are 'Ctrl U', 'U', 'Ctrl' and 'Ctrl+'. These should include `Ctrl`+`U` in the results.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
